hathor/manager.py

```diff
@@ -130,6 +123,7 @@ def __init__(self,
                 'Either enable it, or use the reset-event-queue CLI command to remove all event-related data'
             )
 
+        self.settings = settings
```

Comment:
Other classes use `self._settings` should we do this here too?

Reply:
Makes sense, done in https://github.com/HathorNetwork/hathor-core/pull/787/commits/748afab81c05dbd564a00c5274bacaec0526cc5e